mycore-mods/src/test/java/org/mycore/mods/merger/MCRNameMergerTest.java

```diff
@@ -133,6 +133,14 @@ public void testPreferFamilyGiven() throws JaxenException, IOException {
         MCRMergerTest.test(a, b, b);
     }
 
+    @Test
+    public void testEmptyGiven() throws JaxenException, IOException {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4549276)